source/extensions/omni.isaac.lab/omni/isaac/lab/utils/configclass.py

```diff
@@ -240,6 +241,55 @@ class State:
     cls.__annotations__ = hints
 
 
+def _assert_valid(obj: object, prefix: str = "") -> list:
```

Comment:
NIT: Might be nicer to move this function lower to keep all "init" level things together

```suggestion
def _validate(obj: object, prefix: str = "") -> list[str]:
```